src/components/LoadableComponentErrorBoundary/LoadableComponentErrorBoundary.module.css

```diff
@@ -0,0 +1,97 @@
+@import '../../styles/customMediaQueries.css';
```

Comment:
I don't think that customMediaQueries.css import is needed. 
This is not a separate chunk (the code in components/index.js end up to main-chunk).